blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -81,36 +95,50 @@ public int getIntegerProperty(Enum<? extends ConfigDefaults> name) {
         return getIntegerProperty(name.toString());
     }
```

Comment:
Sorry I'm late to the review. But I have to agree with @ChandraAddala. I don't see much values in adding these static methods (intFromString(), floatFromString()) that simply call the core Java methods. You are also adding tests for each of these static methods -- these don't test any of our Configuration logic.